app/src/main/java/org/fossasia/badgemagic/ui/EditClipartActivity.kt

```diff
@@ -45,4 +48,20 @@ class EditClipartActivity : AppCompatActivity() {
             }
         })
     }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.open_folder, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.open_Folder -> {
+                val intent = Intent(this, DrawerActivity::class.java)
+                intent.putExtra("clipart", "clipart")
```

Comment:
```suggestion
                intent.putExtra("clipart", "clipart")
                intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
```

because we want to cancel the last stacked activities.